bugfix nerf categories + new most_watched + fr dict

framawiki opened the pull request.

framawiki committed:
nerf categories
peer Orlodrim
https://fr.wikipedia.org/wiki/Sujet:Tbtir2yk4moka8f0

framawiki committed:
smallfix frwiki dict + clean duplicates
peer Od1n
https://fr.wikipedia.org/wiki/Sujet:Tbw6h0xnngd6xfu9

framawiki committed:
Bugfix unicode i18n dict #4
need better fix

framawiki committed:
Bugfix unicode i18n dict #4
need better fix

framawiki committed:
add most_watched report

SQL query from MZMcBride https://en.wikipedia.org/w/index.php?title=User_talk:BernsteinBot&diff=762334008&oldid=762110908 (Public domain; bjweeks, MZMcBride; 2009, 2017)

framawiki committed:
add numbers= param

fix

framawiki committed:
add numbers= param

framawiki committed:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	displayTable.py

framawiki committed:
add numbers= param

fix²

framawiki committed:
add numbers= param

fix²²

framawiki committed:
fr dict

framawiki committed:
fr dict

framawiki committed:
fr dict: change titles

Niharika29 commented:
Is this ready for a merge? :)

Niharika29 commented on displayTable.py:
```diff
@@ -5,9 +5,10 @@
 	@param wiki - Language code for the wiki (eg: 'en', 'es', 'it')
 	@param content - Python 2D array (nested list) containing table entries
 	@param desc - Key to the description message of the report
+	@param numbers - If table have to contain numbers of rows
```
Is this param used anywhere?

framawiki replied:
Not yet used.
My main idea was to create an option to allow to add lines number in tables. I've reverted it and for the time being preferred hard code it for each report that need it.

Niharika29 replied:
I still see this in the diff. Better remove it completely and we can add something to automatically add line numbers. That would be simpler.

framawiki replied:
Removed

framawiki committed:
fr dict

https://fr.wikipedia.org/w/index.php?title=Wikip%C3%A9dia:Rapports/%C3%89diteurs_actifs_avec_les_comptes_les_plus_anciens&curid=10300006&diff=134249194&oldid=134184145

framawiki commented:
This is ready to merge.

framawiki committed:
ghost function

Niharika29 commented on main.py:
```diff
@@ -32,9 +32,6 @@ def pagecountbynamespace( self ):
 	def pages_with_most_revisions( self ):
 		self.rep.pages_with_most_revisions()
 
-	def blank_pages( self ):
```
Not sure why this has been dropped. Is the report no longer needed? Sorry I haven't looked at this stuff in a while.

framawiki replied:
I just see that this function do nothing - looks like the rest of this quarry [was removed](https://github.com/Niharika29/database-reports/commit/d0330efad1d5b0407b010b6629986c4e891ca429)

framawiki replied:
But at same time you have written "fixed in BernsteinBot" in remove commit message, is it possible to keep the function, if it's functional, in this repo ?

Niharika29 commented on reports.py:
```diff
@@ -407,6 +407,33 @@ def article_by_size( self ):
+			i += 1
+
+		# Format the data as wikitext
+		text = display_report(self.wiki, content, 'most_watched-desc')
```
( self.wiki, content, 'most-watched-desc' )
Similarly for line below.
Just trying to keep a consistent style here. :)

Niharika29 commented on reports.py:
```diff
@@ -415,7 +442,7 @@ def publish_report( self, title, content ):
 		reports_base_url = dict_obj[ str( 'reports_base_url' ) ]
 		report_title = dict_obj[ str( title ) ]
-		print str( reports_base_url + report_title )
+		print(str( reports_base_url + report_title ))
```
What's the difference?

framawiki replied:
Habits :) removed.

Niharika29 commented on i18n/i18n.py:
```diff
@@ -89,6 +89,13 @@
 		"most_edited_page_last_month-editcount": "Revisions",
+
+		"most_watched-page-title": "Most-watched pages",
+		"most_watched-desc": "Most-watched non-deleted pages (limited to the first 1000 entries)",
```
Non-deleted seems obvious, no?

framawiki replied:
Yes, I've removed this in the English dict, but keep for French : there are few old pages that was deleted but followed by lot of accounts.

framawiki committed:
remove unused documentation

framawiki committed:
cosmetic

framawiki committed:
en dict

framawiki committed:
cleanup

framawiki commented:
Thanks for your review and advice :)

Niharika29 commented:
Hey, @framawiki. Sorry to get back to this so late. Are you still interested in getting this merged?
